bookkeeper-server/src/main/java/org/apache/bookkeeper/zookeeper/ZooKeeperClient.java

```diff
@@ -317,6 +317,7 @@ protected ZooKeeperClient(String connectString,
 
     @Override
     public void close() throws InterruptedException {
+        logger.info("zooKeeper client start to close...");
```

Comment:
no need to log this at "info" level

Reply:
thanks, update log level from info to debug @eolivelli